java-version/accelerator.yaml

```diff
@@ -51,3 +51,13 @@ engine:
         options:
           oldKeyPath: '"$.spec.build.env[?(@.name == ''BP_JVM_VERSION'')].value"'
           value: "#workloadJavaVersion"
+
+    # *.yaml
+    # Targeting Pipeline definitions. Update versions of OpenJdk image.
+    - include: [ "config/*.yaml" ]
+      exclude: [ "config/workload.yaml" ]
+      chain:
+        - type: ReplaceText
+          regex:
+            pattern: "(?<unmodified>image: bellsoft/liberica-openjdk-\\w*:)(\\d*)"
```

Comment:
Wouldn't it be more robust to use `+` instead of `*` (in both places)?

Reply:
Done

Reply:
@ericbottard Please merge